crates/holochain_keystore/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Updated to lair 0.0.3
+  - switch to sqlite/sqlcipher for kestore backing database
+  - enable encryption via passphrase (not on windows)
```

Comment:
could add a description of the breaking changes to the config here

Reply:
:+1: done, added that to the holochain_conductor_api CHANGELOG.